src/main/java/com/synopsys/integration/blackduck/configuration/BlackDuckServerConfigBuilder.java

```diff
@@ -193,14 +185,13 @@ private void validateBlackDuckCredentials(BuilderStatus builderStatus) {
         }
     }
 
-    private void validateBlackDuckURL(BuilderStatus builderStatus) {
+    private void validateBlackDucUrlL(BuilderStatus builderStatus) {
```

Comment:
Typo: extra 'L' at the end of the method name

Reply:
sweet catch!